contributing.md

```diff
@@ -0,0 +1,41 @@
+## Help us to help you!
```

Comment:
For now, you can keep the contributing.md to the minimum (no policy yet about external contributions).

```
# Contributing 

## Development environment
To install the development environment, please follow these steps:
```bash
pip install -r requirements_dev.txt
pre-commit install
pre-commit run --all-files
```